<launch>

    <arg name="calibration" default="$(find svo_ros)/param/hawk/camera0.yaml"/>
    <arg name="image_topic" default="/hawk/camera_0/image_raw"/>

    <node pkg="svo_ros" type="vo" name="svo" clear_params="true" output="screen" ns="hawk">
        <!-- Camera topic to subscribe to -->
        <param name="cam_topic" value="$(arg image_topic)" type="str" />

        <!-- Camera calibration file -->
        <rosparam file="$(arg calibration)"/>

        <!-- Default parameter settings: choose between vo_fast and vo_accurate -->
        <rosparam file="$(find svo_ros)/param/vo_accurate.yaml" />

        <!-- Initial camera orientation, make it point downwards -->
        <param name="init_rx" value="3.14" />
        <param name="init_ry" value="0.00" />
        <param name="init_rz" value="0.00" />
    </node>

</launch>
